src/components/NoteDialogComponents/NoteDialogHeader.scss

```diff
@@ -6,13 +6,14 @@
   color: $color-white;
   width: 100%;
   height: $stack-view__header-height;
+  min-height: 96px;
```

Comment:
<img width="1102" alt="Bildschirmfoto 2022-11-18 um 10 27 23" src="https://user-images.githubusercontent.com/35272402/202668150-9f8b4bf2-5f93-4256-b5df-fa98cfc9bd25.png">
it's being squished at the top otherwise